tests/RequestWatcherTest.php

```diff
@@ -93,4 +98,26 @@ public function testMessageContent()
 
         $this->assertNotEmpty(Telescope::$entriesQueue);
     }
+
+    public function testIgnorePath()
+    {
+        $event = new RequestHandled(Request::create('/test'), new Response());
+
+        $options = config($this->configName);
+
+        new RequestWatcher($options)->recordRequest($event);
```

Comment:
let's make small refactoring and move separate local helper for it OR create watcher instance inside the `setUp` method.

Looks like at the moment there are no reasons to create watcher dynamically as we using static config, changed in setUp, but i not sure

```suggestion
        $this->getRequestWatcher()->recordRequest($event);

        //OR

        $this->requestWatcher->recordRequest($event);
```